sql_generators/active_users_deletion_requests/templates/mobile_deletion_request_query.sql

```diff
@@ -207,8 +208,10 @@ SELECT
     search_with_ads,
     uri_count,
     active_hours_sum,
-    first_seen_date
+    first_seen_date,
+    durations
   ),
+  COUNT(DISTINCT IF(days_since_seen = 0 AND durations > 0, client_id, NULL)) AS qdau,
```

Comment:
Should this be `dau` and the line below be `all_dau`?

Reply:
let's proceed with `daily_users` instead of all_dau (to be changed on mobile_query.sql as well)

Reply:
Ah, right. Also, should we use `dailys_users` in desktop_query.sql and focus_android_query.sql as well?

Reply:
Updated to daily_users! (These are only the users that reported a ping on that date)

Thanks for the note on the `deletion_request` tables. They're still a prototype and only for mobile, updated for consistency.